.github/actions/bpftrace/scripts/check-encryption-leaks.bt

```diff
@@ -136,7 +136,9 @@ kprobe:br_forward
   if ($proto == PROTO_IPV4) {
     if (
       (ENCRYPTION_TYPE_IPSEC && $ip4h->protocol != PROTO_ESP) ||
-      (ENCRYPTION_TYPE_WIREGUARD && !($ip4h->protocol == PROTO_UDP && ($udph->dest == PORT_WIREGUARD || $udph->source == PORT_WIREGUARD)))
+      (ENCRYPTION_TYPE_WIREGUARD && !($ip4h->protocol == PROTO_UDP &&
+                                      (bswap($udph->dest) == PORT_WIREGUARD ||
+                                        bswap($udph->source) == PORT_WIREGUARD)))
```

Comment:
how we didnt' see any issue before? missing `bswap` (aka ntoh) should result in constant errors, no?

Reply:
All wg-encrypted traffic (nodeip <-> nodeip) would enter the first condition:
```c
(str($8) == "wireguard" && !($ip4h->protocol == PROTO_UDP && ($udph->dest == PORT_WIREGUARD || $udph->source == PORT_WIREGUARD)))
```

However, we were not seeing the issue as our leak condition would not match it:
```c
if (($src_is_pod && $dst_is_pod) || ($pod_to_pod_via_proxy && ($src_is_pod || $dst_is_pod))) {
```

Reply:
:+1:  (This is such a stupid bug introduced by me, we should backport this.